### DISCUSSION TITLE
Visualizing iec: entities with apexcharts-card

### DISCUSSION THREAD

**Discussion by smarg120** [GuyKh/iec-custom-component#196] [Q&A] [answered]
i am trying to visualize iec sensors in apexcharts-card since it provides much better UI/UX than built-in HA statistics card....but seems that it cannot recognize the iec:xxx entities......is there any workaround around this?
thanks

**Comment by smarg120** [comment, marked as answer]
right after opening i realized that may be the apexcharts issue...and indeed quick google found that request for support....
https://github.com/RomRider/apexcharts-card/issues/707

**Comment by GuyKh** [comment]
Exactly
